components/plugins/wordpress/WordpressHero.vue

```diff
@@ -0,0 +1,181 @@
+<template>
+  <div
+    class="hero d-flex justify-content-center align-items-center position-relative mx-auto"
+  >
+    <div
+      class="hero__content d-flex flex-column align-items-center my-auto text-center px-3"
+    >
+      <h1 class="hero__heading">
+        <span class="d-block">
+          Embeding
+          <nuxt-img src="/logo.svg" alt="Formester" class="ms-2 formester" />
+        </span>
+        <span class="d-block">
+          on
+          <nuxt-img
+            src="/plugins/wordpress-logo.svg"
+            alt="Wordpress"
+            class="ms-2 wordpress"
+          />
+        </span>
+      </h1>
+      <p class="mt-3">
+        Utilize our intuitive WordPress plugin to integrate Formester forms into
+        your WordPress pages effortlessly.
+      </p>
+      <a
+        href="https://app.formester.com/users/sign_up"
+        class="btn button hero__button mt-4 mb-4 mb-lg-0"
+        >Get Started for Free</a
+      >
+    </div>
+    <div>
+      <transition name="slide-in-left">
+        <nuxt-img
+          src="/plugins/wordpress-hero-left.svg"
+          alt="Hero-Image"
+          class="img-fluid position-absolute hero-image left"
+          v-if="showImage"
+        />
+      </transition>
+      <transition name="slide-in-right">
+        <nuxt-img
+          src="/plugins/wordpress-hero-right.svg"
+          alt="Hero-Image"
+          class="img-fluid position-absolute hero-image right"
+          v-if="showImage"
+        />
+      </transition>
+    </div>
+  </div>
+</template>
+<script>
+import TrustSeals from '@/components/home/TrustSeals.vue'
+export default {
+  components: {
+    TrustSeals,
+  },
+  data() {
+    return {
+      showImage: false,
+    }
+  },
+  mounted() {
+    this.showImage = true
+  },
+}
```

Comment:
what's the use of `showImage` anyways you are making it `true` in mounted and it's not updated anywhere else too.

Reply:
I have used this animation in the hero image.